Cargo.lock

```diff
@@ -1614,7 +1614,7 @@ dependencies = [
 
 [[package]]
 name = "joystream-node-runtime"
-version = "6.14.0"
+version = "6.15.0"
```

Comment:
Lets actually bump the runtime spec version from 15 to 16 and the version in Cargo.toml for the runtime crate to `6.16.0`

Reply:
Ok